guides/source/getting_started.md

```diff
@@ -312,8 +312,6 @@ It should look something like the following:
 Rails.application.routes.draw do
   get 'welcome/index'
 
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-
```

Comment:
True, I think, instead of add we can say replace and mention that we removed it.